age/benches/throughput.rs

```diff
@@ -70,15 +88,14 @@ fn bench(c: &mut Criterion<CyclesPerByte>) {
 #[cfg(unix)]
 criterion_group!(
     name = benches;
-    config = Criterion::default()
-        .with_measurement(CyclesPerByte)
+    config = setup_criterion()
         .with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
     targets = bench
 );
 #[cfg(not(unix))]
 criterion_group!(
     name = benches;
-    config = Criterion::default()
+    config = setup_criterion()
         .with_measurement(CyclesPerByte);
```

Comment:
D'oh, I just realized this line was supposed to be removed. It's redundant but also its presence means that this PR doesn't fix the case of non-Unix non-x86/64 machines (Windows ARM I guess).